test/longrunning.ts

```diff
@@ -207,7 +209,8 @@ describe('longrunning', () => {
                 expect(result).to.deep.eq(RESPONSE_VAL);
                 expect(metadata).to.deep.eq(METADATA_VAL);
                 expect(rawResponse).to.deep.eq(SUCCESSFUL_OP);
-                expect(client.getOperation.callCount).to.eq(0);
+                expect((client.getOperation as sinon.SinonSpy).callCount)
```

Comment:
Is it possible to make `client.getOperation` of type `sinon.SinonSpy` in the source itself?